core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -179,12 +179,8 @@ class KafkaApisTest extends Logging {
     overrideProperties.foreach( p => properties.put(p._1, p._2))
     val config = new KafkaConfig(properties)
 
-    val listenerType = ListenerType.BROKER
-    val enabledApis = ApiKeys.apisForListener(listenerType).asScala
-
     val apiVersionManager = new SimpleApiVersionManager(
```

Comment:
It can be streamlined to 
```java
    val apiVersionManager = new SimpleApiVersionManager(
      ListenerType.BROKER,
      true,
      () => new FinalizedFeatures(MetadataVersion.latestTesting(), Collections.emptyMap[String, java.lang.Short], 0))
```

and then we can remove the constructor 
```java
    public SimpleApiVersionManager(ApiMessageType.ListenerType listenerType,
                                   Features<SupportedVersionRange> brokerFeatures,
                                   boolean enableUnstableLastVersion,
                                   Supplier<FinalizedFeatures> featuresProvider) {
```